include/asterisk/callerid.h

```diff
@@ -199,6 +199,9 @@ int callerid_feed_jp(struct callerid_state *cid, unsigned char *ubuf, int sample
  */
 void callerid_get(struct callerid_state *cid, char **number, char **name, int *flags);
 
+/*! \brief Same as callerid_get, but also retrieve the redirecting reason, if available */
```

Comment:
Can you reproduce the full doxygen block from `callerid_get(...)` here and be sure to document `redirecting` obviously.